scipy/linalg/tests/test_lapack.py

```diff
@@ -1532,3 +1532,122 @@ def test_lapack_documented():
                 name not in names):
             missing.append(name)
     assert missing == [], 'Name(s) missing from lapack.__doc__ or ignore_list'
+
+
+def test_getrf_getrs():
```

Comment:
I didn't see any tests for getrf/getrs elsewhere, and I planned to use these numbers for the tests of getc2/gesc2, so I implemented these. 

In the end, I didn't end up using these numbers for getc2/gesc2.

Reply:
You can also random numbers. Getrf and getrs are. Tested at their higher level python api in linalg module. So no extra tests necessary

Reply:
Ok. I'll remove these then. I used random numbers for the new wrappers.